### PR TITLE
MOSIP-25855

### PR DESCRIPTION
**Pull request by HimajaDhanyamraju2**

### CODE REVIEW

File: authentication/esignet-integration-impl/src/main/java/io/mosip/authentication/esignet/integration/service/IdaAuthenticatorImpl.java [hunk 1]
```diff
@@ -315,12 +343,55 @@ public boolean isSupportedOtpChannel(String channel) {
...
-    public List<KycSigningCertificateData> getAllKycSigningCertificates() {
-        List<KycSigningCertificateData> certs = new ArrayList<>();
-        return certs;
+    public List<KycSigningCertificateData> getAllKycSigningCertificates() throws KycSigningCertificateException {
+    	try {
+    		String authToken = getAuthToken();
+
+            RequestEntity requestEntity = RequestEntity
+                     .get(UriComponentsBuilder.fromUriString(getCertsUrl).queryParam("applicationId", applicationId).queryParam("referenceId", referenceId).build().toUri())
+                     .header(AUTHORIZATION_HEADER_NAME, AUTHORIZATION_HEADER_NAME)
```
Comment (ghost) [inline authentication/esignet-integration-impl/src/main/java/io/mosip/authentication/esignet/integration/service/IdaAuthenticatorImpl.java]: Is this authorization header required?

File: authentication/esignet-integration-impl/src/main/java/io/mosip/authentication/esignet/integration/service/IdaAuthenticatorImpl.java [hunk 2]
```diff
@@ -315,12 +343,55 @@ public boolean isSupportedOtpChannel(String channel) {
...
+    	RequestWrapper<ClientIdSecretKeyRequest> authRequest = new RequestWrapper<>();
+    	authRequest.setRequesttime(LocalDateTime.now());
+    	ClientIdSecretKeyRequest clientIdSecretKeyRequest = new ClientIdSecretKeyRequest(clientId, secretKey, appId);
+    	authRequest.setRequest(clientIdSecretKeyRequest);
+    	
+    	String requestBody = objectMapper.writeValueAsString(authRequest);
+    	RequestEntity requestEntity = RequestEntity
+                 .post(UriComponentsBuilder.fromUriString(authTokenUrl).build().toUri())
+                 .contentType(MediaType.APPLICATION_JSON)
+                 .header(AUTHORIZATION_HEADER_NAME, AUTHORIZATION_HEADER_NAME)
```
Comment (ghost) [inline authentication/esignet-integration-impl/src/main/java/io/mosip/authentication/esignet/integration/service/IdaAuthenticatorImpl.java]: Is this authorization header required?

Comment (HimajaDhanyamraju2) [reply authentication/esignet-integration-impl/src/main/java/io/mosip/authentication/esignet/integration/service/IdaAuthenticatorImpl.java]: Checked and removed the header.

File: authentication/esignet-integration-impl/src/main/java/io/mosip/authentication/esignet/integration/service/IdaAuthenticatorImpl.java (hunk 1 above)
Comment (HimajaDhanyamraju2) [reply authentication/esignet-integration-impl/src/main/java/io/mosip/authentication/esignet/integration/service/IdaAuthenticatorImpl.java]: Checked and removed the header.

**Pull request merged**